Error while starting the Micro integrator in Windows

**Description:**
We are getting the following error while starting the WSO2 EI 6.4.0 Micro Integrator from the pack in Windows 10. 

```
JAVA_HOME environment variable is set to C:\Program Files\Java\jdk1.8.0_201
CARBON_HOME environment variable is set to C:\PROGRA~1\WSO2\ENTERP~1\64BE80~1.0\wso2\MICRO-~1\bin\..
log4j:ERROR Could not instantiate class [org.wso2.carbon.micro.integrator.core.logging.CarbonMemoryAppender].
java.lang.ClassNotFoundException: org.wso2.carbon.micro.integrator.core.logging.CarbonMemoryAppender cannot be found by org.wso2.carbon.logging_4.4.36
        at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:501)
        at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:421)
        at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:412)
        at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at java.lang.Class.forName0(Native Method)
        at java.lang.Class.forName(Class.java:264)
        at org.apache.log4j.helpers.Loader.loadClass(Loader.java:198)
        at org.apache.log4j.helpers.OptionConverter.instantiateByClassName(OptionConverter.java:327)
        at org.apache.log4j.helpers.OptionConverter.instantiateByKey(OptionConverter.java:124)
        at org.apache.log4j.PropertyConfigurator.parseAppender(PropertyConfigurator.java:785)
        at org.apache.log4j.PropertyConfigurator.parseCategory(PropertyConfigurator.java:768)
        at org.apache.log4j.PropertyConfigurator.configureRootCategory(PropertyConfigurator.java:648)
        at org.apache.log4j.PropertyConfigurator.doConfigure(PropertyConfigurator.java:514)
        at org.apache.log4j.PropertyConfigurator.doConfigure(PropertyConfigurator.java:580)
        at org.apache.log4j.helpers.OptionConverter.selectAndConfigure(OptionConverter.java:526)
        at org.apache.log4j.LogManager.<clinit>(LogManager.java:127)
        at org.apache.log4j.Logger.getLogger(Logger.java:104)
        at org.apache.commons.logging.impl.Log4JLogger.getLogger(Log4JLogger.java:262)
        at org.apache.commons.logging.impl.Log4JLogger.<init>(Log4JLogger.java:108)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at org.apache.commons.logging.impl.LogFactoryImpl.createLogFromClass(LogFactoryImpl.java:1025)
        at org.apache.commons.logging.impl.LogFactoryImpl.discoverLogImplementation(LogFactoryImpl.java:844)
        at org.apache.commons.logging.impl.LogFactoryImpl.newInstance(LogFactoryImpl.java:541)
        at org.apache.commons.logging.impl.LogFactoryImpl.getInstance(LogFactoryImpl.java:292)
        at org.apache.commons.logging.impl.LogFactoryImpl.getInstance(LogFactoryImpl.java:269)
        at org.apache.commons.logging.LogFactory.getLog(LogFactory.java:655)
        at org.wso2.andes.wso2.internal.QpidBundleActivator.<clinit>(QpidBundleActivator.java:33)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at java.lang.Class.newInstance(Class.java:442)
        at org.eclipse.osgi.framework.internal.core.AbstractBundle.loadBundleActivator(AbstractBundle.java:167)
        at org.eclipse.osgi.framework.internal.core.BundleContextImpl.start(BundleContextImpl.java:679)
        at org.eclipse.osgi.framework.internal.core.BundleHost.startWorker(BundleHost.java:381)
        at org.eclipse.osgi.framework.internal.core.AbstractBundle.resume(AbstractBundle.java:390)
        at org.eclipse.osgi.framework.internal.core.Framework.resumeBundle(Framework.java:1176)
        at org.eclipse.osgi.framework.internal.core.StartLevelManager.resumeBundles(StartLevelManager.java:559)
        at org.eclipse.osgi.framework.internal.core.StartLevelManager.resumeBundles(StartLevelManager.java:544)
        at org.eclipse.osgi.framework.internal.core.StartLevelManager.incFWSL(StartLevelManager.java:457)
        at org.eclipse.osgi.framework.internal.core.StartLevelManager.doSetStartLevel(StartLevelManager.java:243)
        at org.eclipse.osgi.framework.internal.core.StartLevelManager.dispatchEvent(StartLevelManager.java:438)
        at org.eclipse.osgi.framework.internal.core.StartLevelManager.dispatchEvent(StartLevelManager.java:1)
        at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
        at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)
log4j:ERROR Could not instantiate appender named "CARBON_MEMORY".
[2019-02-21 11:52:34,451] [micro-integrator]  INFO - QpidBundleActivator Setting BundleContext in PluginManager
[2019-02-21 11:52:35,053] [micro-integrator]  INFO - CarbonCoreActivator Starting WSO2 Carbon...
[2019-02-21 11:52:35,055] [micro-integrator]  INFO - CarbonCoreActivator Operating System : Windows 10 10.0, amd64
[2019-02-21 11:52:35,057] [micro-integrator]  INFO - CarbonCoreActivator Java Home        : C:\Program Files\Java\jdk1.8.0_201\jre
[2019-02-21 11:52:35,060] [micro-integrator]  INFO - CarbonCoreActivator Java Version     : 1.8.0_201
[2019-02-21 11:52:35,063] [micro-integrator]  INFO - CarbonCoreActivator Java VM          : Java HotSpot(TM) 64-Bit Server VM 25.201-b09,Oracle Corporation
[2019-02-21 11:52:35,071] [micro-integrator]  INFO - CarbonCoreActivator Carbon Home      : C:\PROGRA~1\WSO2\ENTERP~1\64BE80~1.0\wso2\MICRO-~1\bin\..
[2019-02-21 11:52:35,077] [micro-integrator]  INFO - CarbonCoreActivator Java Temp Dir    : C:\PROGRA~1\WSO2\ENTERP~1\64BE80~1.0\wso2\MICRO-~1\bin\..\tmp
[2019-02-21 11:52:35,080] [micro-integrator]  INFO - CarbonCoreActivator User             : wso2, en-GB, Asia/Colombo
[2019-02-21 11:52:35,176] [micro-integrator]  INFO - DefaultCryptoProviderComponent 'CryptoService.Secret' property has not been set. 'org.wso2.carbon.crypto.provider.SymmetricKeyInternalCryptoProvider' won't be registered as an internal crypto provider. Please set the secret if the provider needs to be registered.
[2019-02-21 11:52:35,634] [micro-integrator] ERROR - CloudServiceConfigParser Error in building the cloud service configuration.
java.io.FileNotFoundException: C:\PROGRA~1\WSO2\ENTERP~1\64BE80~1.0\wso2\MICRO-~1\bin\..\conf\multitenancy\cloud-services-desc.xml (The system cannot find the path specified)
        at java.io.FileInputStream.open0(Native Method)
        at java.io.FileInputStream.open(FileInputStream.java:195)
        at java.io.FileInputStream.<init>(FileInputStream.java:138)
        at java.io.FileInputStream.<init>(FileInputStream.java:93)
        at org.wso2.carbon.stratos.common.config.CloudServiceConfigParser.loadCloudServicesConfiguration(CloudServiceConfigParser.java:36)
        at org.wso2.carbon.ui.menu.stratos.Util.readManagerURLFromConfig(Util.java:51)
        at org.wso2.carbon.ui.menu.stratos.internal.StratosUIGeneralMenuDSComponent.activate(StratosUIGeneralMenuDSComponent.java:37)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.eclipse.equinox.internal.ds.model.ServiceComponent.activate(ServiceComponent.java:260)
        at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.activate(ServiceComponentProp.java:146)
        at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.build(ServiceComponentProp.java:345)
        at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponent(InstanceProcess.java:620)
        at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponents(InstanceProcess.java:197)
        at org.eclipse.equinox.internal.ds.Resolver.buildNewlySatisfied(Resolver.java:473)
        at org.eclipse.equinox.internal.ds.Resolver.enableComponents(Resolver.java:217)
        at org.eclipse.equinox.internal.ds.SCRManager.performWork(SCRManager.java:816)
        at org.eclipse.equinox.internal.ds.SCRManager$QueuedJob.dispatch(SCRManager.java:783)
        at org.eclipse.equinox.internal.ds.WorkThread.run(WorkThread.java:89)
        at java.lang.Thread.run(Thread.java:748)
[2019-02-21 11:52:35,682] [micro-integrator] ERROR - Util Error while reading the Manager Homepage URL from cloud service configuration.
java.lang.Exception: Error in building the cloud service configuration.
        at org.wso2.carbon.stratos.common.config.CloudServiceConfigParser.loadCloudServicesConfiguration(CloudServiceConfigParser.java:41)
        at org.wso2.carbon.ui.menu.stratos.Util.readManagerURLFromConfig(Util.java:51)
        at org.wso2.carbon.ui.menu.stratos.internal.StratosUIGeneralMenuDSComponent.activate(StratosUIGeneralMenuDSComponent.java:37)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.eclipse.equinox.internal.ds.model.ServiceComponent.activate(ServiceComponent.java:260)
        at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.activate(ServiceComponentProp.java:146)
        at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.build(ServiceComponentProp.java:345)
        at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponent(InstanceProcess.java:620)
        at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponents(InstanceProcess.java:197)
        at org.eclipse.equinox.internal.ds.Resolver.buildNewlySatisfied(Resolver.java:473)
        at org.eclipse.equinox.internal.ds.Resolver.enableComponents(Resolver.java:217)
        at org.eclipse.equinox.internal.ds.SCRManager.performWork(SCRManager.java:816)
        at org.eclipse.equinox.internal.ds.SCRManager$QueuedJob.dispatch(SCRManager.java:783)
        at org.eclipse.equinox.internal.ds.WorkThread.run(WorkThread.java:89)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.FileNotFoundException: C:\PROGRA~1\WSO2\ENTERP~1\64BE80~1.0\wso2\MICRO-~1\bin\..\conf\multitenancy\cloud-services-desc.xml (The system cannot find the path specified)
        at java.io.FileInputStream.open0(Native Method)
        at java.io.FileInputStream.open(FileInputStream.java:195)
        at java.io.FileInputStream.<init>(FileInputStream.java:138)
        at java.io.FileInputStream.<init>(FileInputStream.java:93)
        at org.wso2.carbon.stratos.common.config.CloudServiceConfigParser.loadCloudServicesConfiguration(CloudServiceConfigParser.java:36)
        ... 17 more

```

**Suggested Labels:**
<!-- Optional comma separated list of suggested labels. Non committers can’t assign labels to issues, so this will help issue creators who are not a committer to suggest possible labels-->

**Suggested Assignees:**
<!--Optional comma separated list of suggested team members who should attend the issue. Non committers can’t assign issues to assignees, so this will help issue creators who are not a committer to suggest possible assignees-->

**Affected Product Version:**
WSO2 EI 6.4.0
**OS, DB, other environment details and versions:**    
Windows 10 64 bit
Java 1.8.0_201

**Steps to reproduce:**
1. Get WSO2 EI 6.4.0 zip.
2. Unzip the pack.
3. Run the Micro Integrator profile.

**Related Issues:**
https://github.com/wso2/product-ei/issues/3223

--- a/pom.xml
+++ b/pom.xml
@@ -1531,7 +1531,7 @@
         <org.apache.httpcomponents.wso2.version>4.3.6.wso2v2</org.apache.httpcomponents.wso2.version>
 
         <!-- Carbon mediation version -->
-        <carbon.mediation.version>4.6.106</carbon.mediation.version>
+        <carbon.mediation.version>4.6.108</carbon.mediation.version>
 
         <!-- Carbon kernel version-->
         <carbon.kernel.version>4.4.36</carbon.kernel.version>

